packages/linux-drivers/openvfd/scripts/openvfd-config

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
+
+if [ ! -e /proc/device-tree/openvfd/compatible -a -e /flash/dtb.img ]; then
+  mount -o remount,rw /flash
+    fdtput -p -t s /flash/dtb.img /openvfd status "okay"
+    fdtput -p -t s /flash/dtb.img /openvfd dev_name "openvfd"
+    fdtput -p -t s /flash/dtb.img /openvfd compatible "open,vfd"
+  mount -o remount,ro /flash
+fi
+
+if [ -f /storage/.config/vfd.conf ]; then
+  cp /storage/.config/vfd.conf /run/libreelec/openvfd.conf
+elif [ -f /storage/.config/openvfd.conf ]; then
```

Comment:
vfd configuration has never been stored in /storage/.config/openvfd.conf or /etc/openvfd.conf on any LE community/CE build afaik